recipes/ms-gsl/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "ms-gsl" %}
+{% set version = "4.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/microsoft/GSL/archive/v{{ version }}.tar.gz
+  sha256: f0e32cb10654fea91ad56bde89170d78cfbf4363ee0b01d8f097de2ba49f6ce9
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+    - ninja
+
+test:
+  requires:
+    - cmake
+    - {{ compiler('cxx') }}
+    - ninja
+  files:
+    - test
+
+  commands:
```

```suggestion
  commands:
    - test -f ${PREFIX}/include/gsl/gsl  # [unix]
    - if not exist %PREFIX%\\Library\\include\\gsl\\gsl exit 1  # [win]
    - test -f $PREFIX/share/cmake/Microsoft.GSL/Microsoft.GSLConfig.cmake  # [unix]
    - if not exist %PREFIX%\\Library\\share\\cmake\\Microsoft.GSL\\Microsoft.GSLConfig.cmake exit 1 # [win]

```
These tests make it obvious that this is a header-only library by only looking at the recipe.